container.go

```diff
@@ -305,6 +305,8 @@ func (c *ContainerRequest) validateContextOrImageIsSpecified() error {
 	return nil
 }
 
+// validateMounts ensures that the mounts do not have duplicate targets.
+// It will check the Mounts and HostConfigModifier.Binds fields.
 func (c *ContainerRequest) validateMounts() error {
```

Comment:
TBH I think this validation function is very little useful, as it's does really too few checks.

We probably need a follow up discussion to remove it or rethink what to check here